packages/valaxy/src/client/locales/zh-CN.yml

```diff
@@ -59,9 +60,9 @@ footer:
   total_visitors: 总访客量
 
 counter:
-  archives: 暂无日志 | 共计 {count} 篇日志
-  categories: 暂无分类 | 共计 {count} 个分类
-  tags: 暂无标签 | 共计 {count} 个标签
+  archives: 暂无日志 | 共计 1 篇日志 | 共计 {count} 篇日志
```

Comment:
1 post has already been counted in the `count`, the modification here is not required.

Reply:
![}L)5H@J7B@A~0R~MNTYX)RR](https://user-images.githubusercontent.com/54099603/173386933-94e17dff-a36c-456d-bf75-69086f3d91fb.png)
![``NA%SVRH`4LL``J)2S4FUJ](https://user-images.githubusercontent.com/54099603/173386944-07b64e2d-43b0-4cf2-b159-abb0dfc19df7.png)
![RM 2FNF$5{AASLQ2%}3DN14](https://user-images.githubusercontent.com/54099603/173386956-1decee64-dfaf-46b6-9c09-2c3c9a7b8ae5.png)
![F {$N3)~MJ{XO_QCJB54VEO](https://user-images.githubusercontent.com/54099603/173386982-fe71c410-f566-46c0-80e3-c75ef6482534.png)

I'm confused.